ruby/Rakefile

```diff
@@ -37,10 +37,12 @@ proto2_protos = %w[
   google/protobuf/compiler/plugin.proto
 ]
 
-if system('../src/protoc --version')
-  protoc_command = '../src/protoc' 
+if system('../bazel-bin/protoc --version')
```

Comment:
This should be fine, but it might be more consistent overall to check the environment, maybe like:

if ENV['PROTOC'] != '':
  protoc_command = ENV['PROTOC']

(This seems fine, too, though.)

Reply:
Good idea, done.